apps/api/package.json

```diff
@@ -15,10 +15,11 @@
 		"migrate": "pnpm drizzle-kit migrate",
 		"start:prod": "pnpm migrate && tsx ./src/main.js",
 		"test": "tsx --test --test-reporter=spec --experimental-test-coverage ./tests/**/*.test.ts",
-		"test:local": "pnpm run test:api && pnpm run test:db && pnpm run test:services",
+		"test:local": "pnpm run test:api && pnpm run test:db && pnpm run test:services && pnpm run test:functions",
 		"test:api": "tsx --test --test-reporter=spec --experimental-test-coverage ./tests/api/*.test.ts",
 		"test:db": "tsx --test --test-reporter=spec --experimental-test-coverage ./tests/db/*.test.ts",
-		"test:services": "tsx --test --test-reporter=spec --experimental-test-coverage ./tests/services/*.test.ts"
+		"test:services": "tsx --test --test-reporter=spec --experimental-test-coverage ./tests/services/*.test.ts",
+		"test:functions": "tsx --test --test-reporter=spec --experimental-test-coverage ./tests/functions/*.test.ts"
```

Comment:
In the test updates PR I would like to use a `test:regex` approach rather than continually adding individual scripts